pandas/tests/indexes/test_indexing.py

```diff
@@ -66,6 +66,10 @@ def test_take(self, index):
             with pytest.raises(AttributeError, match=msg):
                 index.freq
 
+        scalar_index = 1
+        with pytest.raises(TypeError):
```

Comment:
i think you will have to add the `match` parameter with the error msg. And maybe also point back to the issue.

Reply:
`match` param should be in `pytest.raises`. Take a look at how they are setup in other tests

Reply:
this is still incorrect

Reply:
and maybe the msg should say that it can't be scalar (which you are actually checking)

Reply:
try also with a 2D array